src/SDKs/IotHub/Management.IotHub/Generated/IotHubResourceOperations.cs

```diff
@@ -2532,23 +2531,23 @@ internal IotHubResourceOperations(IotHubClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<IotHubNameAvailabilityInfo>> CheckNameAvailabilityWithHttpMessagesAsync(OperationInputs operationInputs, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<IPage<EndpointHealthData>>> GetEndpointHealthWithHttpMessagesAsync(string resourceGroupName, string iotHubName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This looks like a breaking change. Please update the package version to `3.0.0`

Reply:
Done